themes/default/layouts/page/product.html

```diff
@@ -111,53 +122,185 @@ <h5>{{ $feature.title }}</h5>
         </div>
     </section>
 
-    <section id="stats" class="text-center lg:mt-32 lg:pb-32 relative">
-        <div class="shape-background product-stats-section">
+    <section class="container mx-auto text-center relative">
+        <div class="mx-auto card bg-white w-full my-4 lg:my-0 text-center p-6 lg:p-8 mx-0 lg:mx-4 z-10 flex flex-row items-center">
+            <div class="w-1/2 pr-20">
+                <h4>Build infrastructure automation into your software and applications</h4>
+            </div>
+            <div class="w-full flex flex-row">
+                <div class="flex flex-col bg-white w-full my-4 lg:my-0 text-center p-6 mx-0 z-10">
+                    <img class="h-12" src="/images/product/altana.png" alt="Altana logo" />
```

Comment:
is there a way to make this logo smaller? it looks much bigger than the other two next to it